makefile

```diff
@@ -69,6 +69,10 @@ GIT_HASH = $(shell git rev-parse --verify HEAD)
 COMMON_CFLAGS += -DHASH=\"$(GIT_HASH)\"
 endif
 
+ifdef APP_PATH
```

Comment:
Where does APP_PATH comes from ? When is it defined and when is it not ?

Reply:
You can define it at make evocation. if not, there is default behavior i tried to be exactly what was before to not break anything.